### DISCUSSION TITLE
Show current line scrollable popup through click

### DISCUSSION THREAD

**Discussion by TheColorRed** [gitkraken/vscode-gitlens#1912] [Feedback]
I have the current line `scrollable` set to false so the popup doesn't display when I mouse over an annotation. Is there a way to make it so that when I click on an annotation that it is shown, maybe through the code actions lightbulb on the left hand side of the editor, or a <kbd>ctrl</kbd> + <kbd>click</kbd> on the annotation itself?

```json
{
  "gitlens.currentLine.scrollable": false
}
```

**Comment by eamodio** [comment]
`"gitlens.currentLine.scrollable": false` only stops the annotation from taking up space in the document, so that you can scroll horizontally to see the annotation. It was largely added to deal with VS Code issues with end of line decorators (which is what the GitLens annotation is) when word-wrap is enabled. And since VS Code doesn't provide a way for GitLens to tell that word-wrap is enabled or not, the setting was born.

If you want to control the hover that shows on the annotation, you can customize its behavior here:

![image](https://user-images.githubusercontent.com/641685/158689709-04668d38-340d-43bd-8ceb-1a134183a5b3.png)

Unfortunately there isn't really a way for GitLens to trigger a hover on anything but an actual hover. I say "really" because VS Code does have a command that can be run to show the hover for the place where the cursor is, but that won't provide enough to make it really work.
